netmaster/objApi/apiController.go

```diff
@@ -2325,3 +2330,17 @@ func validatePorts(ports []string) bool {
 	}
 	return true
 }
+
+func validateGlobalConfig(netmode string) error {
+	globalConfig := contivModel.FindGlobal("global")
+	if globalConfig.FwdMode == "" {
```

Comment:
nil check ?

Reply:
i don't think `globalConfig` would be `nil` once the netmaster started